packages/wallet/frontend/src/pages/transfer/send.tsx

```diff
@@ -26,13 +26,19 @@ import {
 } from '@/utils/constants'
 import { useOnboardingContext } from '@/lib/context/onboarding'
 import { QuoteDialog } from '@/components/dialogs/QuoteDialog'
+import { assetService, Rates } from '@/lib/api/asset'
 
 type SendProps = InferGetServerSidePropsType<typeof getServerSideProps>
 
 const SendPage: NextPageWithLayout<SendProps> = ({ accounts }) => {
   const [openDialog, closeDialog] = useDialog()
   const [paymentPointers, setPaymentPointers] = useState<SelectOption[]>([])
   const [balance, setBalance] = useState('')
+  const [currentExchangeRates, setCurrentExchangeRates] = useState<Rates>()
+  const [selectedAssetCode, setSelectedAssetCode] = useState('')
+  const [receiverAssetCode, setReceiverAssetCode] = useState('')
+  const [exchangeCurrencyText, setExchangeCurrencyText] = useState('')
```

Comment:
I don't know what to say about having another state action for displaying some text. I think we can do this with conditional rendering.